charts/scroll-proving-sdk/values.yaml

```diff
@@ -0,0 +1,57 @@
+---
+global:
+  nameOverride: &app_name scroll-proving-sdk
+  fullnameOverride: *app_name
+
+image:
+  repository: scrolltech/sdk-cloud-prover
+  pullPolicy: Always
+  tag: sindri-2791edca
+
+command:
+  - "/bin/sh"
+  - "-c"
+  - "exec cloud --config /sdk_prover/config.json"
+
+resources:
+  requests:
+    memory: "100Mi"
+    cpu: "50m"
+  limits:
+    memory: "500Mi"
+    cpu: "100m"
+
+persistence:
+  *app_name:
+    enabled: true
+    type: configMap
+    mountPath: /sdk_prover/
+    name: scroll-proving-sdk-config
+
+service:
+  main:
+    enabled: true
+    ports:
+      http:
+        enabled: true
+        port: 80
+
+defaultProbes: &default_probes
+  enabled: true
+  custom: true
+  spec:
+    httpGet:
+      path: "/"
+      port: 80
+
+probes:
+  liveness:
+    <<: *default_probes
+  readiness:
+    <<: *default_probes
+  startup:
+    <<: *default_probes
+
+# Add here the config.json file. You can see an example in example-config.yaml
```

Comment:
Should this be in production.yaml since every operator will use it? Or example-config.yaml could be in /values/ and we recommend just passing both in the helm upgrade -i command? We'd need to modify example-config.yaml to add `scrollConfig: |`